auditeval/test.go

```diff
@@ -159,6 +159,7 @@ func getFlagValue(s, flag string) string {
 	var flagVal string
 	pttns := []string{
 		flag + `\s*=\s*"(.*)"`,
+		flag + `\s*=\s*(.*)`,
```

Comment:
```suggestion
		flag + `\s*=\s*([^\s]*)`,
```
This should capture anything after the = and optional whitespace, until there is some whitespace. I think it means the next line is unnecessary.